src/commondao/common.dao.model.ts

```diff
@@ -168,6 +178,11 @@ export interface CommonDaoOptions extends CommonDBOptions {
    */
   anonymize?: boolean
 
+  /**
+   * @default false (for streams). Setting to true enables deletion of immutable objects
+   */
+  overrideImmutability?: boolean
```

Comment:
I suggest to call it `allowMutability`, as "allow" explicitly states that it's allowed, while "override" says that it overrides the default behavior, but to know the default behavior you need to check the docs.